src/xpi.js

```diff
@@ -0,0 +1,80 @@
+import yauzl from 'yauzl';
+
+
+export default class Xpi {
+
+  constructor(filename, zipLib=yauzl) {
+    this.filename = filename;
+    this.zipLib = zipLib;
+    this.metadata = null;
```

Comment:
Heh, actually I guess that makes sense too :-)

Cool, let's stick with null then. Totally makes sense in my brain semantically now :-)

- tofumatt

> On 25 Sep 2015, at 09:15, Stuart Colville <notifications@github.com> wrote:
> 
> In src/xpi.js:
> 
> > @@ -0,0 +1,80 @@
> > +import yauzl from 'yauzl';
> > +
> > +
> > +export default class Xpi {
> > +
> > +  constructor(filename, zipLib=yauzl) {
> > +    this.filename = filename;
> > +    this.zipLib = zipLib;
> > +    this.metadata = null;
> Yeah this is all about intent. I chose to use null because it's not 'undefined' if that makes sense?
> 
> —
> Reply to this email directly or view it on GitHub.
>